src/flamenco/runtime/context/fd_exec_slot_ctx.c

```diff
@@ -169,7 +169,7 @@ fd_exec_slot_ctx_recover_( fd_exec_slot_ctx_t *   slot_ctx,
   fd_memcpy(&slot_bank->banks_hash, &oldbank->hash, sizeof(oldbank->hash));
   fd_memcpy(&slot_bank->fee_rate_governor, &oldbank->fee_rate_governor, sizeof(oldbank->fee_rate_governor));
   slot_bank->lamports_per_signature = oldbank->fee_calculator.lamports_per_signature;
-  slot_ctx->prev_lamports_per_signature = oldbank->fee_calculator.lamports_per_signature;
+  slot_ctx->prev_lamports_per_signature = manifest->lamports_per_signature;
   if( oldbank->hashes_per_tick )
```

Comment:
This seems an odd change.  Why did you move where we got lamports_per_signature?  It should have been copied into the old bank when we first booted?

Reply:
writing this while it says fresh in case we come back to this pr:
the bank and the manifest has a field for lamports per signature.